### PR TITLE
[WAGON-501] ScmGitExeWagonTest

### PR DESCRIPTION
**Pull request by basinilya**

### CODE REVIEW

File: wagon-providers/wagon-scm/src/test/java/org/apache/maven/wagon/providers/scm/AbstractScmGitWagonTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+        File origRepo = getTestFile( "target/test-classes/test-repo-git" );
+
+        File testRepo = getTestFile( "target/test-classes/test-repo-git-test" );
+
+        FileUtils.deleteDirectory( testRepo );
+
+        FileUtils.copyDirectoryStructure( origRepo, testRepo );
+
+        repository = "scm:git:file://" + testRepo.toURI().getPath();
```
Comment (michael-o) [inline wagon-providers/wagon-scm/src/test/java/org/apache/maven/wagon/providers/scm/AbstractScmGitWagonTest.java]: Note that `File#toURI()` produces an invalid URI. Convert to NIO Path first and then to URI.

Comment (basinilya) [reply wagon-providers/wagon-scm/src/test/java/org/apache/maven/wagon/providers/scm/AbstractScmGitWagonTest.java]: > File#toURI()` produces an invalid URI
Can you give me a little more info on that?

Comment (michael-o) [reply wagon-providers/wagon-scm/src/test/java/org/apache/maven/wagon/providers/scm/AbstractScmGitWagonTest.java]: See https://issues.apache.org/jira/browse/MNG-6386

Comment (basinilya) [reply wagon-providers/wagon-scm/src/test/java/org/apache/maven/wagon/providers/scm/AbstractScmGitWagonTest.java]: done.
I guess AbstractScmSvnWagonTest can also be fixed like that.

**Comment by jira-importer** [comment]
Resolve #566

**Pull request closed without merging**